OpenTelemetry.sln

```diff
@@ -267,6 +267,7 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared", "Shared", "{A49299FB-C5CD-4E0E-B7E1-B7867BBD67CC}"
 	ProjectSection(SolutionItems) = preProject
 		src\Shared\ActivityInstrumentationHelper.cs = src\Shared\ActivityInstrumentationHelper.cs
+		src\OpenTelemetry\Internal\HttpSemanticConventionHelper.cs = src\OpenTelemetry\Internal\HttpSemanticConventionHelper.cs
```

Comment:
Why is this pointing to `src\OpenTelemetry\Internal` instead of `src\OpenTelemetry\Shared`?

Reply:
wad added by VS. Fixed.